compiler.events-only.toml

```diff
@@ -21,9 +21,9 @@ uid = "1705072206827"
 title = "steering: P-high Issue Review"
 description = "Continue going over our P-high tagged issues"
 location = "#t-compiler/meetings on Zulip (https://rust-lang.zulipchat.com/#narrow/stream/238009-t-compiler.2Fmeetings/)"
-last_modified_on = "2024-03-14T15:46:00.00Z"
-start = { date = "2024-01-19T10:00:00.00", timezone = "America/New_York" }
-end = { date = "2024-01-19T11:00:00.00", timezone = "America/New_York" }
+last_modified_on = "2024-11-08T16:36:20.00Z"
```

Comment:
Might want to update this before merging.  It seems it might be better for this to be too far in the future than too far in the past with respect to how some clients seem to handle this.

Reply:
Oh, this is interesting and unexpected, why is it a fake future date for the `last_modified_on` needed?
Probably also good to document this quirk in the README. What's a good `last_modified_on` value when adding or editing an event? Would it be okay close to the beginning of the event?

thanks

Reply:
My understanding is that some clients use this in a way similar to the HTTP `Last-Modified` header in deciding whether they can short-circuit updating an event or recurrence.  So if that timestamp predates when we publish the updated `ical` file, the client may miss updates to the event.

Usually I set it far enough in the future so as to be sure it will not be in the past when the update is merged and CI completes.  Agreed it would be good to write something about this in the README.

Reply:
OK @traviscross I've updated all `last_modified_on` (just to be sure) to 3 months in the future. Added a smallnote in the READMe (hope I've interpreted correctly your explaination).

Reply:
Thanks.  Probably I was thinking a few days rather than three months.  Hopefully it doesn't take us that long to merge PRs :)

Reply:
ok, updated docs and fields (+10 days in the future)